### PR TITLE
Forbid creation of cert subject mappings with the same subject

### PR DESCRIPTION
**Pull request by la4ezar**
**Description**
Currently, it is possible to create cert subject mappings with the same subjects if their delimiter is different. For example, instead of comma(',') you use plus sign('+') for delimiter. This should be forbidden and this PR does that.

Changes proposed in this pull request:
- Forbid creation of cert subject mappings with the same subject

**Related issue(s)**

- ...

**Pull Request status**

- [ ] Implementation
- [ ] Unit tests
- [ ] Integration tests
- [ ] `chart/compass/values.yaml` is updated 
- [ ] Mocks are regenerated, using the automated script

### CODE REVIEW

**Review by StanislavStefanov — DISMISSED** [review]

**Review by StanislavStefanov — DISMISSED** [review]

File: chart/compass/values.yaml [hunk 1]
```diff
@@ -139,7 +139,7 @@ global:
       name: compass-pairing-adapter
     director:
       dir: dev/incubator/
-      version: "PR-3170"
+      version: "3177"
```
Comment (dzahariev) [inline chart/compass/values.yaml:142]: Please Add "PR-" here.

Comment (la4ezar) [reply chart/compass/values.yaml]: Good catch!

**Review by StanislavStefanov — APPROVED** [review]

**Pull request merged**